eth_account/typed_transactions/blob_transactions/blob_transaction.py

```diff
@@ -332,10 +330,15 @@ def payload(self) -> bytes:
         """
         Returns this transaction's payload as bytes.
 
-        Here, the TransactionPayload = rlp([chainId,
-        nonce, maxPriorityFeePerGas, maxFeePerGas, gasLimit, to, value, data,
-        accessList, maxFeePerBlobGas, blobVersionedHashes, signatureYParity, signatureR,
-        signatureS])
+        Here, the transaction payload looks like:
+
+        .. code-block:: python
+
+            TransactionPayload = rlp([chainId,
```

Comment:
Same as the previous comment here. This could be left in the docstring I think since ``rlp`` doesn't work exactly like this.